phpseclib/Crypt/Blowfish.php

```diff
@@ -534,15 +519,14 @@ function _setupInlineCrypt()
         $lambda_functions =& Crypt_Blowfish::_getLambdaFunctions();
 
         // We create max. 10 hi-optimized code for memory reason. Means: For each $key one ultra fast inline-crypt function.
+        // (Currently, for Crypt_Blowfish, one generated $lambda_function cost on php5.5@32bit ~100kb unfreeable mem and ~180kb on php5.5@64bit)
         // After that, we'll still create very fast optimized code but not the hi-ultimative code, for each $mode one.
-        $gen_hi_opt_code = (bool)( count($lambda_functions) < 10);
-
-        switch (true) {
-            case $gen_hi_opt_code:
-                $code_hash = md5(str_pad("Crypt_Blowfish, {$this->mode}, ", 32, "\0") . $this->key);
-                break;
-            default:
-                $code_hash = "Crypt_Blowfish, {$this->mode}";
+        $gen_hi_opt_code = (bool)( count($lambda_functions) < 10 );
+
+        // Generation of a uniqe hash for our generated code
```

Comment:
I'm not sure what you mean by that? FWIW some of this stuff was in petrich's orig PR. Although this PR is based off of that I can't necessarily give immediate insight into why everything was done the way it is - in some cases I'd have to do research or in some cases it might have been just arbitrary too. So for some of the stuff you mentioned I'll have to look into it before I comment on it.